docs/integreatly/modules/ROOT/pages/_partials/data-sync/index-1.js

```diff
@@ -1,33 +0,0 @@
-const express = require('express')
-//Include our server libraries
-const { VoyagerServer, gql } = require('@aerogear/voyager-server')
-
-//Provide your graphql schema
-const typeDefs = gql`
-  type Query {
-    hello: String
-  }
-`
-
-//Create the resolvers for your schema
-const resolvers = {
-  Query: {
-    hello: (obj, args, context, info) => {
-      return `Hello world`
-    }
-  }
-}
-
-//Initialize the library with your Graphql information
-const apolloServer = VoyagerServer({
-  typeDefs,
-  resolvers
-})
-
-//Connect the server to express
-const app = express()
-apolloServer.applyMiddleware({ app })
-
-app.listen(4000, () =>
-  console.log(`🚀 Server ready at http://localhost:4000/graphql`)
-)
```

Comment:
not sure why this PR would delete this file?

Reply:
Does this have to stay in partials or can it be in the pages folder?

Reply:
it should stay in _partials (cos it's not 'content' per se), however, I dunno, maybe it would work in pages

Reply:
works in asciidoctor
![image](https://user-images.githubusercontent.com/5154224/70902689-44351700-1ff5-11ea-925d-65e19ca6e11c.png)

doesn't work in antora
![image](https://user-images.githubusercontent.com/5154224/70902748-5616ba00-1ff5-11ea-84b6-916388c08225.png)